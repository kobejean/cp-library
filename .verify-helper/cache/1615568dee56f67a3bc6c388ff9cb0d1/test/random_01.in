5
120999146 136987924 40280999 438515315 805152546 234164454 129099933 971852321 983937488 410134225 668461222 574343409 885417013 394300887 86086437 570981511 221329455 57893312 584381871 154204049 738660729 728257729 551666498 540440394 165573287 512342480 452470821 669622703 340240729 965382636 953292241 144902419
86035524 289571086 987645468 423893886 928104871 464748969 105872436 123582647 242780928 505538017 199537013 745991283 983136444 65304379 842202106 646596193 140145001 630412665 134706415 170189559 917006962 498351627 78228861 784928508 545206283 276287905 127534371 580361498 602636875 833497706 569752320 168745739
