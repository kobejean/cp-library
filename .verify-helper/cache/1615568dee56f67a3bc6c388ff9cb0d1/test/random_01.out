752338355 369667436 387532326 461549619 292201508 13377087 406050718 908695460 593036470 209403180 841685153 877825495 348111378 312469941 997611530 810204318 856616864 63222311 735711248 607193826 498182016 439499720 945648912 495563992 881569477 847058873 854259198 872452401 148070992 885065201 29353850 354244519
